src/pages/index.js

```diff
@@ -82,6 +84,7 @@ class IndexPage extends React.Component {
           <VContainer>
             <TopAuthors pull_requests={this.state.pull_requests} />
             <TopIssues issues={this.state.issues} />
+            <Authors authors={this.state.pull_requests} />
```

Comment:
Would be better to have it as a separate page entirely.
I would like to have a link, like three dots..., from the top authors card that sends to the authors page